starboard/shared/starboard/loader_app_metrics.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "starboard/shared/starboard/loader_app_metrics.h"
+
+#include "starboard/extension/loader_app_metrics.h"
+
+namespace starboard {
+namespace shared {
+namespace starboard {
+
+namespace {
+
+static CrashpadInstallationStatus g_crashpad_installation_status;
```

Comment:
Please add a comment explaining why this global variable doesn't need thread safety protection.

Reply:
I added comments to the extension's interface (header) and implementation.

Please let me know what you think. I could see the argument for removing the single-thread assumption and implementing the extension with mutual exclusion now, in version 1 - for example, in case some refactoring in Cobalt caused this extension's accessors to be called from some new thread.

Reply:
I think it is OK as long as we document it.